src/core.rs

```diff
@@ -71,7 +73,7 @@ impl Core {
             // or require a raw output (like the `wc` command).
             inner_flags.layout = Layout::OneLine;
 
-            flags.should_quote = false;
```

Comment:
how about changing `should_quote` to `literal`?

Reply:
Changed!